modules/monitoring-setting-log-levels-for-monitoring-components.adoc

```diff
@@ -0,0 +1,129 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+[id="setting-log-levels-for-monitoring-components_{context}"]
+= Setting log levels for monitoring components
+
+You can configure the log level for Prometheus Operator, Prometheus, Alertmanager and Thanos Ruler.
+
+The following log levels can be applied to each component in the `cluster-monitoring-config` and `user-workload-monitoring-config` ConfigMaps:
+
+.Available log levels by monitoring component
+[options="header"]
+|====
+|Component |cluster-monitoring-config ConfigMap |user-workload-monitoring-config ConfigMap
+|Prometheus Operator |`debug`, `info`, `warn`, and `error` |`debug`, `info`, `warn`, and `error`
+|Prometheus |`all`, `debug`, `info`, `warn`, `error`, and `none` |`all`, `debug`, `info`, `warn`, `error`, and `none`
```

Comment:
default loglevel is info, there is not `none` loglevel

Reply:
Thank you for highlighting this. Since the log levels are the same then for all four components, I have removed the table from the module. I will push my changes to the fork shortly.

I have also now specified the default log level in the module.

Reply:
@lilic can you please review this module? Thank you!

Reply:
The only think probably worth mentioning here is the fact that these log levels do not touch client-go's internal log level, else LGTM. But this information is already very very implementation specific so we might as well leave it out.